docs/deprecated-blocks.md

```diff
@@ -0,0 +1,86 @@
+# Deprecated Blocks
+
+When updating static blocks markup and attributes, block authors need to consider existing posts using the old versions of their block. In order to provide a good upgrade path, you can choose one of the following strategies:
+
+ - Do not deprecate the block and create a new one (a different name)
+ - Provide a "deprecated" version of the block allowing users opening these blocks in Gutenberg to edit them using the updated block.
+
+A block can have several deprecated versions, Gutenberg will try them one after another until finding the one that matches the saved markup.
+
+To declare a "deprecated version", you need to copy the old `attributes`, `support` and `save` properties from the previous old definition to the `deprecated` property of the udpated block.
+
+### Example:
+
+{% codetabs %}
+{% ES5 %}
+```js
+var el = wp.element.createElement,
+	registerBlockType = wp.blocks.registerBlockType;
+
+registerBlockType( 'gutenberg/block-with-deprecated-version', {
+
+	// ... other block properties go here
+
+	attributes: {
```

Comment:
Should we have two examples, one using the same set of attributes even with new markup?

Reply:
Sure, I added an example with `migrate` as well